.travis.yml

```diff
@@ -1,8 +1,7 @@
 sudo: false
 language: go
 go:
-  - 1.5
-  - 1.6
+  - 1.7
```

Comment:
I removed 1.6 because I switched `golang.org/x/net/context` to `context` and 1.6 was understandably failing. But this is a good point. I reverted the package change and added 1.6 back now.